deployments/helm/nvidia-dra-driver-gpu/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Does it make sense to symlink to the `LICENSE` file in the root? I suppose their contents are the same.

Reply:
I have confirmes that when making the following changes:
```
 git status
On branch jp/helm-apply-license
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        new file:   NOTICE

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        typechange: deployments/helm/nvidia-dra-driver-gpu/LICENSE
        typechange: deployments/helm/nvidia-dra-driver-gpu/NOTICE
```
and generating the package using `hack/package-helm-charts.sh` the resultant tar does contain the `LICENSE` and `NOTICE` files with the expected content.

Reply:
I like this ++

Reply:
Thanks for thinking about linking without blinking

:) okäse :cheese: pushed a commit.

I thought about this (symlinking) when making this patch.  Thoughts were: in this case, cost of duplication is ~zero. I also wanted to quickly get over with it and thought that it might even be smart to keep things decoupled. The most important part is that the tarball contains the proper files. From the git repo point of view we can choose not to care (is a symlinked license an actual license? what would the supreme court say? would elon come to our rescue? is armageddon about to happen soon?).